Sources/WordPressSharedObjC/Utility/WPFontManager.m

```diff
@@ -125,4 +120,20 @@ + (void)loadFontResourceNamed:(NSString *)name withExtension:(NSString *)extensi
     return;
 }
 
++ (NSBundle *)resourceBundle {
+#if SWIFT_PACKAGE
+    return SWIFTPM_MODULE_BUNDLE;
+#else
+    // When installed via CocoaPods, the resources will be bundled under `WordPressShared.bundle`.
+    // This follows the implementation from `NSBundle+WordPressShared`.
+    NSBundle *defaultBundle = [NSBundle bundleForClass:[self class]];
+    NSURL *sharedBundleURL = [defaultBundle.bundleURL URLByAppendingPathComponent:@"WordPressShared.bundle"];
```

Comment:
The Swift version [uses `resourceURL`](https://github.com/wordpress-mobile/WordPress-iOS-Shared/blob/3defd5e1a0b96e3cce0ee83163746563d48c6490/Sources/WordPressShared/Utility/NSBundle%2BWordPressShared.swift#L19), instead of `bundleURL`, should we keep them consistent?

Reply:
Good catch. This is addressed in https://github.com/wordpress-mobile/WordPress-iOS-Shared/pull/331/commits/623fc90c6ae9c8a4b1dd96d4379e0a18cb4cb415 🙂